### PR TITLE
fixes #204 - fop:environment:setup-dev works fine on docker install.

### LINKED ISSUE #204: fop:dev:setup-env fails because of type error
#### Describe the bug
Capture d’écran de 2021-11-09 17-50-47

### Fop Console Version
1.4.0

### Prestashop Version
1.7.6.1

### Php Version
php version 7.2

### PR DESCRIPTION
**Pull request by SebSept**
`(new \PrestaShop\PrestaShop\Adapter\Debug\DebugMode())->enable()` is supposed to return an int but it can also return `null`.
I didn't search why but that's a fact.

To reproduce the problem, install PrestaShop with docker and can reproduce the issue.
https://github.com/PrestaShop/docker/blob/master/base/config_files/defines_custom.inc.php
Without docker, you can try with this setup (not tested) : when a defines_custom.inc.php file is present and the environment variable PS_DEV_MODE is set.

I also found that `(new \PrestaShop\PrestaShop\Adapter\Debug\DebugMode())->isDebugModeEnabled()` return an wrong value when using PS_DEV_MODE env variable.
Define _PS_DEV_MODE_ in config.inc.php to false and set PS_DEV_MODE=1, having a `defines_custom.inc.php` file.
In this case, the shop runs in debug mode, but `(new \PrestaShop\PrestaShop\Adapter\Debug\DebugMode())->isDebugModeEnabled()` will return false.
This behavior can be considered as a bug. I'll open an issue on PrestaShop.

----

- In short, the command doesn't crash anymore on docker ( [defines_custom.inc.php](https://github.com/PrestaShop/docker/blob/master/base/config_files/defines_custom.inc.php) present and PS_DEV_MODE set)
- it tells the problem and how to switch the shop to debug mode.

### CODE REVIEW

**Review by Kaudaj — APPROVED** [review]
Nice catch! Does this happen for `fop:environment:debug` too?
Maybe you could later move this logic to `debug` command and [call it](https://symfony.com/doc/current/console/calling_commands.html) from `setup-dev` one, to avoid code duplication?

**Comment by SebSept** [comment]
Yes, it's the same for fop:environment:debug
We should add an issue, I forgot. 
And yes, we could maybe refactorize, but  that's maybe not possible.

**Comment by Kaudaj** [comment]
Ok LGTM! 👍

**Comment by SebSept** [comment]
issue created #225
